lib/licensed/source/pip.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+require "json"
+require "English"
+
+module Licensed
+  module Source
+    class Pip
+      def initialize(config)
+        @config = config
+      end
+
+      def type
+        "python"
+      end
+
+      def enabled?
+        @config.enabled?(type) && File.exist?(@config.pwd.join("requirements.txt"))
+      end
+
+      def dependencies
+        packages = parse_requirements_txt
+
+        @dependencies = packages.map do |package_name|
```

Comment:
please use `||=` and call `parse_requirements_text` inline so the dependencies will be cached

`@dependencies ||= parse_requirements_txt.map do |package_name|`

Reply:
👍